gitlab/v4/objects/issues.py

```diff
@@ -98,10 +95,7 @@ class GroupIssueManager(ListMixin, RESTManager):
         "updated_after",
         "updated_before",
     )
-    _types = {
-        "iids": types.CommaSeparatedListAttribute,
-        "labels": types.CommaSeparatedListAttribute,
-    }
+    _types = {"iids": types.ArrayAttribute, "labels": types.CommaSeparatedListAttribute}
```

Comment:
lol, this is infuriating:
https://docs.gitlab.com/ee/api/issues.html#list-issues
```
GET /issues?iids[]=42&iids[]=43
GET /issues?labels=foo,bar
```

:grin: thanks for going through all the endpoints. I hope the API docs are accurate.